src/test/scala/magellan/OsmSuite.scala

```diff
@@ -0,0 +1,56 @@
+package magellan
+
```

Comment:
I like these unit tests, could we also add one unit test where an OSM format file is parsed and read ? I don't know much about this format, but maybe mapzen.com has a file that we can use : ideally a fairly small file that we can check in (there are similar small files in src/test/resources for other formats)